docs/config.md

```diff
@@ -5,15 +5,27 @@ These are the config settings available to jj/Jujutsu.
 
 ## Config files and TOML
 
-The config settings are loaded from the following locations. Less common ways to
-specify `jj` config settings are discussed in a later section.
+`jj` loads several types of config settings:
 
-* [The user config file]
-* `.jj/repo/config.toml` (per-repository)
+- The default settings. These cannot be edited.
 
-See the [TOML site] and the [syntax guide] for a description of the syntax.
+- The user settings. These can be edited with `jj config edit --user`. User
+settings are located in [the user config file], which can be found with `jj
+config path --user`.
 
-[The user config file]: #user-config-file
+- The repo settings. These can be edited with `jj config edit --repo` and are
+located in `.jj/repo/config.toml`.
+
+- Settings [specified in the command-line](#specifying-config-on-the-command-line).
+
+These are listed in the order they are loaded; the config listed earlier would
+be overridden by the config listed later if they disagree. Every type of config
```

Comment:
I would rewrite the first sentence in the present tense instead of using a future tense. So "They are listed in the order they are loaded; the config listed earlier will be overriden ..."